packages/devtools/lib/src/settings/settings_controller.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:vm_service/vm_service.dart';
+
+import '../globals.dart';
+
+class SettingsController {
+  SettingsController({this.onFlagListReady, this.onIsAnyFlutterAppReady});
+
+  final Function(FlagList) onFlagListReady;
```

Comment:
rather than onIsAnyFlutterAppReady make this
`Function(String)` onSdkVerionChange
which could return:
`${isAnyFlutterApp ? 'Flutter' : 'Dart'} SDK Version: '`
Keeping the logic in the controller where it is easier to test.

Reply:
I thought of this earlier but was unsure, will do! That's a good point :D